packages/apollo/lib/drivers/apollo-errors.driver.ts

```diff
@@ -0,0 +1,47 @@
+import { ASTNode, GraphQLError, Source, SourceLocation } from 'graphql';
+import { ApolloServerErrorCode } from '@apollo/server/errors';
+
+export class ApolloError extends Error implements GraphQLError {
+  locations: readonly SourceLocation[];
+  path: readonly (string | number)[];
+  nodes: readonly ASTNode[];
+  source: Source;
+  positions: readonly number[];
+  originalError: Error;
+  constructor(
+    public message: string,
+    public extensions: Record<string, any>,
+    public code?: string,){
+    super(message);
+
+    if (!this.name) {
+      Object.defineProperty(this, 'name', { value: 'ApolloError' });
+    }
+
+    this.extensions = { ...extensions, code };
+  }
+}
+
+export class UserInputError extends ApolloError {
```

Comment:
Why do we need these classes to be included in this package?

Reply:
If you want to keep backwards compatibility only.